README.md

```diff
@@ -18,12 +20,25 @@ This Gitbook serves as a place to capture the discussions as they evolve and for
 This effort and output is not binding and does not form a final decision on governance implementation. Any final implementation of a governance structure or process would require a full on-chain stake-weighted vote by validators.
 {% endhint %}
 
+## Scope
+
 Initially, this document will begin by exploring some key questions:
 
 * Why - why do we need governance
-* What - what is governance, what areas, actions and policies does it cover
 * Who - who participates in governance, in which roles
+* What - what is governance, what areas, actions and policies does it cover
 * How - how do we implement
 
+## Principles
+
+TBD - Suggested in TG chat
+
+## Contributors
```

Comment:
Github automatically displays this info (https://github.com/laine-sa/solana-governance-think-tank/graphs/contributors) wouldn't it be simpler than manually updating the README over time?

Reply:
I think it would be nice to have a list in the Gitbook as my sense is many people who review the Gitbook won't look at the source.

Reply:
Yeah but then just link to the Github page maybe?

Reply:
We could but I don't see the harm in listing contributors directly in the Gitbook and linking to the contributors page. I'll add the link and if you'd prefer to remove the list from the Gitbook, please go ahead and do that.

Reply:
Thanks let's leave this as is for now, like I initially mentioned my concern is just maintenance if the contributor list grows